libs/MeshKernel/tests/src/Mesh2DTest.cpp

```diff
@@ -510,18 +505,22 @@ TEST(Mesh2D, DeleteHangingEdge)
     std::vector<meshkernel::Point> nodes;
```

Comment:
since there is no hanging edge delete is this test relevant? or should be renamed to something else?

Reply:
The hanging edge is added a few lines after the mesh has been constructed. I have added a comment